addons/material-forum-editor-buttons/icons.css

```diff
@@ -0,0 +1,147 @@
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_bold/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .markItUpButton2 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_italic/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .markItUpButton3 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_underlined/v5/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+
+.markItUp .markItUpButton4 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/strikethrough_s/v6/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+
+.markItUp .markItUpButton5 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/insert_photo/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .markItUpButton6 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/insert_link/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+
+.markItUp .markItUpButton7 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_size/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .markItUpButton8 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_list_bulleted/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .markItUpButton9 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_list_numbered/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .markItUpButton10 a {
+    background-image: url("https://gist.githubusercontent.com/easrng/4f63ce27f1970632528143430698db89/raw/f20856e9c4ab9ea176207f4980556c3fa4ec87b4/listitem.svg") !important;
+    background-size: 20px;
+}
+
+.markItUp .markItUpButton11 a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/format_quote/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .markItUpButton12 > a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/insert_emoticon/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .browser-os-button a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/computer/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .clean a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/code/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+.markItUp .preview a {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/done/v4/24px.svg?download=true") !important;
+    background-size: 20px;
+}
+#ibbBtn, #uploadButton {
+    background-image: url("https://fonts.gstatic.com/s/i/materialicons/cloud_upload/v4/24px.svg?download=true") !important;
```

Comment:
isn't there already a pretty good one for image-uploader?

Reply:
No